src/frontend/mame/ui/swlist.h

```diff
@@ -52,24 +52,23 @@ class menu_software_list : public menu
 	virtual void handle() override;
 
 private:
-	struct entry_info {
-		entry_info *next;
-
-		const char *short_name;
-		const char *long_name;
+	struct entry_info
```

Comment:
Please tell the compiler that you want move constructor/assignment operator generated to speed up operations when used in a vector (see `menu_item` for an example).